### PR TITLE
Provide UI feedback for failed actions in the taker

### PR DESCRIPTION
**Pull request by klochowicz**
Also:
- improve the handling on the maker side.
- populate errors created within CfdTable

### CODE REVIEW

**Comment by klochowicz** [comment]
I have tested it in the UI open new order request from the taker and on maker's acceptance (both with temporarily adding an `anyhow::bail()`

File: daemon/src/routes_taker.rs [hunk 1]
```diff
@@ -120,37 +129,37 @@ pub async fn post_cfd_action(
...
         | CfdAction::AcceptRollOver
         | CfdAction::RejectRollOver => {
-            return Err(status::BadRequest(None));
-        }
-        CfdAction::Commit => {
-            cfd_action_channel
-                .do_send(Commit { order_id: id })
-                .map_err(|e| status::BadRequest(Some(e.to_string())))?;
+            return Err(HttpApiProblem::new(StatusCode::BAD_REQUEST)
+                .detail("taker cannot invoke this actions"));
```
Comment (da-kami) [inline daemon/src/routes_taker.rs]:
```suggestion
                .detail(format!("taker cannot invoke action {}", action)));
```

Comment (da-kami) [reply daemon/src/routes_taker.rs]: not super important - it's one of these "we should never see this" errors :D

**Review by da-kami — APPROVED** [review]

**Pull request merged**